monitoring/ceph-mixin/requirements-alerts.txt

```diff
@@ -1,2 +1 @@
-pyyaml
```

Comment:
Rather than deleting this dependency, which is required, I think it would be better to pin the version to 6.0, like so: `pyyaml==6.0`. I tried this on my install-deps.sh script, and it made it so there are not two conflicting pyyaml versions.

Reply:
Done.